.gitignore

```
@@ -0,0 +1,2 @@
+__pycache__
+.env
```

From my comment before

```suggestion
__pycache__
.*
```